src/engine.rs

```diff
@@ -1012,17 +1038,19 @@ impl<'a, 'b> Cpu<'a, 'b> {
                         self.hartid, self.state.pc, addr
                     );
                 }
-                trace!(
-                    "Store 0x{:x} = 0x{:x} if 0x{:x} ({}B)",
+                debug!(
```

Comment:
Same here.

Reply:
Addressed in [this commit](https://github.com/pulp-platform/banshee/pull/18/commits/95619028583909da22c571e6fdbf3cd400d35545).